repository-data/webfiles/web/index.jsp

```diff
@@ -0,0 +1,10 @@
+
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+  <head>
+    <title>$Title$</title>
+  </head>
+  <body>
+  $END$
+  </body>
+</html>
```

Comment:
What does this file do?

Reply:
I am not too sure, I think it may have got generated when I was looking at this, https://documentation.bloomreach.com/14/library/development/develop-with-intellij.html

I have now removed the said folder and file.